block-producer/src/store/mod.rs

```diff
@@ -93,6 +97,8 @@ impl ApplyBlock for DefaultStore {
 
 #[async_trait]
 impl Store for DefaultStore {
+    #[allow(clippy::blocks_in_conditions)] // Workaround of `instrument` issue
+    #[instrument(skip(self), ret, err, fields(COMPONENT))]
     async fn get_tx_inputs(
         &self,
         proven_tx: SharedProvenTx,
```

Comment:
Similar comments to the ones before. But this one we can probably keep at `info` level.